packages/@blockprotocol/type-system-node/tests/uri.test.ts

```diff
@@ -0,0 +1,51 @@
+import { isValidVersionedUri, parseBaseUri } from "..";
```

Comment:
It' usually `test` instead of `tests`, but we can keep the folder name as is for now.

Reply:
I'll fix on the next PR, thanks :)